src/aws/osml/image_processing/sar_complex_imageop.py

```diff
@@ -0,0 +1,157 @@
+import logging
+from typing import Optional, Tuple
+
+import numpy as np
+
+TWO_PI = np.pi * 2.0
+
+logger = logging.getLogger(__name__)
+
+
+def image_pixels_to_complex(
+    image_pixels: np.ndarray, pixel_type: Optional[str] = None, amplitude_table: Optional[np.typing.ArrayLike] = None
```

Comment:
Just a note - you don't have to do = None when you specify the Optional Typings

Reply:
I think you're confusing an the optional type hint with an optional vs. positional parameter; those are two different things. You do need to provide the default value or else callers need to provide the second None. See example below:

    >>> from typing import Optional
    >>> def foo(a:int, b:Optional[int]) -> None:
    ...        print(a, b)
    >>> foo(1)
    Traceback (most recent call last):
          File "<stdin>", line 1, in <module>
          TypeError: foo() missing 1 required positional argument: 'b'
    >>> def bar(a:int, b:Optional[int] = None) -> None:
    ...        print(a, b)
    >>> bar(1)
    1 None